SmallCommitsWorkshopTests/Controllers/FizzBuzzControllerTests.cs

```diff
@@ -16,12 +21,50 @@ internal sealed class FizzBuzzControllerTests {
 				.Setup( x => x.Calculate( number ) )
 				.Returns( calculatedValue );
 
-			FizzBuzzController sut = new FizzBuzzController( fizzBuzzService: fizzBuzzService.Object );
+			FizzBuzzController sut = new FizzBuzzController(
+				fizzBuzzService: fizzBuzzService.Object,
+				usersContext: null
+			);
 
 			Assert.AreEqual(
 				expected: calculatedValue,
 				actual: sut.Get( number: number )
 			);
 		}
+
+		[Test]
+		public async Task GetForUser_CallsCalculate() {

```

Comment:
This technically is an API (integration) test?

Reply:
How so?

Reply:
There is an instance of the server and DB context below. It's technically talking to the "database".

Reply:
Oh, yes, you are correct.  Sorry, I had this confused with `FizzBuzzServiceTests` .

Do you know how to Mock this stuff?  I tried a couple of approaches and they all failed :(.

Reply:
I moved the test in with the other `FizzBuzz` integration tests.

Reply:
I do not know how to mock the DB context either.